### PR TITLE
adds Definition of Done

### LINKED ISSUE #143: Write a Definition-of-Done
for: 
- [x] issues
- [x] milestones

### PR DESCRIPTION
**Pull request by JanEberlein**
closes #143

### CODE REVIEW

File: docs/Definition-of-Done.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Definition of Done
+
+## Issues
+
+An issue is finished successfully when:
+- all of the issue's contents are solved
+- the changes are pushed to its branch (created when assigning yourself)
+- a pull request to master is opened and successfully closed:
+ - all PR-checks successful (build, linting, test, type-check)
+ - the changes have been reviewed and approved by two reviewers
```
Comment (tcoro) [inline docs/Definition-of-Done.md:10]: maybe this is better?
```suggestion
 - the changes have been reviewed and approved by atleast two reviewers
```

File: docs/Definition-of-Done.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Definition of Done
+
+## Issues
+
+An issue is finished successfully when:
+- all of the issue's contents are solved
+- the changes are pushed to its branch (created when assigning yourself)
```
Comment (tcoro) [inline docs/Definition-of-Done.md:7]:
```suggestion
- the changes are pushed to its branch (automated on first issue assignment)
```

File: docs/Definition-of-Done.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- a pull request to master is opened and successfully closed:
+ - all PR-checks successful (build, linting, test, type-check)
+ - the changes have been reviewed and approved by two reviewers
+ - all open conversations have been resolved
+
+Only then can the issue be closed (can be done via PR).
```
Comment (tcoro) [inline docs/Definition-of-Done.md:13]:
```suggestion
The issue can be closed with a PR after completing all steps above.
```

File: docs/Definition-of-Done.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Milestones
+
+A milestone was finished successfully when:
```
Comment (tcoro) [inline docs/Definition-of-Done.md:17]: you are using past tense here, and then present in the points afterwards

```suggestion
A milestone is finished successfully when:
```

**Review by tcoro — CHANGES_REQUESTED** [review]

File: docs/Definition-of-Done.md (hunk 1 above)
Comment (JanEberlein) [reply docs/Definition-of-Done.md:10]: yes 😄

**Review by tcoro — APPROVED** [review]

File: docs/Definition-of-Done.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+- a pull request to master is opened and successfully closed:
+ - all PR-checks successful (build, linting, test, type-check)
+ - the changes have been reviewed and approved by at least two reviewers
+ - all open conversations have been resolved
+
+The issue can be closed with a PR after completing all steps above.
+
+## Milestones
+
+A milestone is finished successfully when:
```
Comment (anbraten) [inline docs/Definition-of-Done.md:17]:
```suggestion
If following points apply, a milestone can be treated as done:
```

**Review by anbraten — CHANGES_REQUESTED** [review]

File: docs/Definition-of-Done.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+# Definition of Done
+
+## Issues
+
+An issue is finished successfully when:
+- all of the issue's contents are solved
+- the changes are pushed to its branch (automated on first issue assignment)
```
Comment (JanEberlein) [inline docs/Definition-of-Done.md:7]:
```suggestion
- the changes are pushed to its branch (automated on first issue assignment)
- the changes are documented sufficiently
```

**Review by anbraten — APPROVED** [review]

**Pull request merged**